packages/types/src/Metadata/v2/MetadataRegistry.ts

```diff
@@ -261,11 +261,7 @@ export class Metadata$H512 extends Null {
 }
 
 export class FieldName$Unnamed extends U16 {}
-export class FieldName$Named extends Text {
-  toString (): string {
```

Comment:
This is causing issue on codec because `encodedLength` depends on result of `toU8a` which depends on `toJSON` which depends on `toString`